ui/src/routes/_layout/organizations/$orgId/index.tsx

```diff
@@ -58,34 +58,72 @@ import { paginationSchema } from '@/schemas';
 import { LastJobStatus } from './products/$productId/-components/last-job-status';
 import { LastRunDate } from './products/$productId/-components/last-run-date';
 import { LastRunStatus } from './products/$productId/-components/last-run-status';
+import { TotalRuns } from './products/$productId/-components/total-runs';
 
 const defaultPageSize = 10;
 
-const columns: ColumnDef<Product>[] = [
-  {
-    accessorKey: 'product',
-    header: () => <div>Products</div>,
-    cell: ({ row }) => (
-      <>
-        <Link
-          className='block font-semibold text-blue-400 hover:underline'
-          to={`/organizations/$orgId/products/$productId`}
-          params={{
-            orgId: row.original.organizationId.toString(),
-            productId: row.original.id.toString(),
-          }}
-        >
-          {row.original.name}
-        </Link>
-        <div className='text-sm text-muted-foreground md:inline'>
-          {row.original.description}
-        </div>
-      </>
-    ),
-  },
-  {
-    accessorKey: 'runStatus',
-    header: () => <div>Last Run Status</div>,
+const columnHelper = createColumnHelper<Product>();
+
+// In anticipation of these column definitions to be changed later, when the corresponding
+// endpoint is implemented, columnHelper.accessor() is only used when the data being
+// shown contains data from the column helper type.
+
+const columns = [
+  columnHelper.accessor(
+    ({ name, description }) => {
+      return name + description;
+    },
```

Comment:
Did you find something that made you choose to leave this function like this? I'm still a bit confused if now hypothetically you would add client side sorting to this table (I know we probably won't), would the value based on which the sorting would be done be now `name+description`. Cause then we would probably only want `name` here.

Reply:
For sorting it probably doesn't affect anything, as the name is the first element, so alphabetically there's no difference with unique names and name + description. Could maybe communicate the intent a little clearer with just the name, and same goes for the url + type.

Another aspect if filtering. There could be a use for having the description in the filterable value, as then you'd be able to include information in the description in the filter. But either probably works, and we can fine-tune it when filtering is in-place.

So both work IMO and are fine by me.